charts/opennms-operator/values.yaml

```diff
@@ -12,4 +12,18 @@ operator:
       memory: 50Mi
 customErrors:
   image: quay.io/kubernetes-ingress-controller/custom-error-pages-amd64:0.4
-  debug: true
\ No newline at end of file
+  debug: true
+ssl:
+  enabled: true
+  issuer:
+    issuerServer: https://acme-v02.api.letsencrypt.org/directory
+    issuerUserEmail: email@address.com
```

Comment:
Is this valid for a user email?

Reply:
No it isn't. All of these values are just placeholders to be filled in by whomever uses the operator's helm charts.